apps/full-site-editing/full-site-editing-plugin/blocks/post-content/index.php

```diff
@@ -1,10 +1,18 @@
 <?php
 
 function render_post_content_block( $attributes, $content ) {
```

Comment:
I found that I needed to add something like this to prevent an infinite loop while saving:

```php
// early return to avoid infinite loops in the REST API
if ( is_admin() || ( defined( 'REST_REQUEST' ) && REST_REQUEST ) ) {
	return $content;
}
```

Reply:
resolved by @Copons on b3feca48dff0425ebd1c543c1e5fe7e4043b8f9f. 🚀